proposals/0317-enforce-32-data-shreds.md

```diff
@@ -0,0 +1,55 @@
+---
+simd: '0317'
+title: Enforce 32 data + 32 coding shreds
+authors:
+  - Emanuele Cesena
+category: Standard
+type: Core
+status: Review
+created: 2025-07-08
+feature:
+---
+
+## Summary
+
+All clients currently send 32 data + 32 coding shreds, however Turbine still allows
+to send and receive a variable number of shreds. With this change we will enforce
+sending and receiving only 32 data + 32 coding shreds.
+
+## Motivation
+
+It is inconvinient to support many combinations of data + coding shreds.
+There's no security benefit, and in fact security is reduced when the number of
+shreds is low.
```

Comment:
maybe elaborate on the security benefit? can reference https://github.com/solana-foundation/solana-improvement-documents/pull/204/files#diff-e9a92f44cc9e8d1789705b2a4a51bba175df745484c6638e475514d14eefe716R146

basically equivocation detection is simplified, because you have to see the same shred idx to see the equivocation (vs. before you could have overlapping windows)

Reply:
thanks a lot, I've included all 3 comments